02.calendar/cal.rb

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env ruby
+require "optparse"
+require "date"
+
+opt = OptionParser.new
+option = {}
+opt.on( "-m ARG", desc = "set month variable" ){ |m| option[:m] = m }
+opt.on( "-y ARG", desc = "set year variable" ){ |y| option[:y] = y }
+opt.parse(ARGV)
+
+if option[:m] != nil
```

Comment:
Rubyだと慣習として以下のように書くことが多いですね。

```suggestion
if option[:m]
```

あと、これは好みになりますが、三項演算子を使うのも一手です。

```ruby
month = foo ? bar : baz
```

Reply:
・falseまたはnilであれば偽。
・それ以外はすべて真。
チェリー本p.36を復習しました。「!= nil」よく間違えて使ってしまうので気をつけます。

三項演算子に書き直してみたとろだいぶ見た目がスッキリしました。チェリー本p.67にも書いていましたが、
シンプルなif分だと三項演算子の方がスッキリかけますね。